ale/rotation.py

```diff
@@ -110,7 +110,7 @@ def __mul__(self, other):
             new_rot = self._rot * other._rot
             return ConstantRotation(new_rot.as_quat(), other.source, self.dest)
         elif isinstance(other, TimeDependentRotation):
-            return TimeDependentRotation((self._rot * other._rots).as_quat(), other.times, other.source, self.dest, av=self._rot.apply(other.av))
+            return TimeDependentRotation((self._rot * other._rots).as_quat(), other.times, other.source, self.dest, av=other.av)
```

Comment:
This and the other changes in rotation are because NAIF defines the angular velocity as always being in the source reference frame. See the detailed output in the docs for [xf2rav](https://naif.jpl.nasa.gov/pub/naif/toolkit_docs/FORTRAN/spicelib/xf2rav.html)

Reply:
We are heavily tied to NAIF for ISIS. No reason to change that now, maybe in 6.0.